lib/alertwildfire_API.py

```diff
@@ -65,22 +68,45 @@ def get_individual_camera_info(cameraID):
 def request_current_image(outputDir, cameraID):
     camera_info = get_individual_camera_info(cameraID)
     if camera_info["image"]["time"]:
-        timeStamp = camera["image"]["time"]###need to convert their format
+        timeStamp = camera_info["image"]["time"]###need to convert their format
+
+
+        logging.warning('not yet implemented camera_info/image/time')
+        timeStamp = time.mktime(datetime.datetime.now().timetuple())
```

Comment:
Please document what's wrong with the original code.  Are you aware of a camera ID that provides image/time?  Same question for position/time below.

Reply:
its only a tag so that the program will alert us when they update the capability of recording the timestamp. the current implementation of code if triggered by their update will cause our code to fail since the the format of their implementation of camera_info["*"]["time"] will likely not be consistent with ours.  this allows for the program to run normally in the future if they update before we can respond. it merely takes the else case until we can adjust.